src/main/groovy/com/readytalk/gradle/js/tasks/AsarPack.groovy

```diff
@@ -0,0 +1,23 @@
+package com.readytalk.gradle.js.tasks
+
+import com.moowork.gradle.node.task.NodeTask
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.OutputFile
+
+class AsarPack extends NodeTask {
+  @InputDirectory
+  def File inputDir
+
+  @OutputFile
+  def File outputFile
+
+  @Override
+  void exec() {
+    outputFile.parentFile.mkdirs();
+
+    setScript(project.file('node_modules/asar/bin/asar'))
```

Comment:
What if the asar dependency isn't installed? Does this assume the user of this project has setup their package.json with the correct dependency?